lib0/src/any.rs

```diff
@@ -73,19 +74,13 @@ impl Into<Any> for f32 {
 
 impl Into<Any> for u32 {
     fn into(self) -> Any {
-        Any::BigInt(self as i64)
-    }
-}
-
-impl Into<Any> for i64 {
-    fn into(self) -> Any {
-        Any::BigInt(self)
+        Any::Number(self as f64)
```

Comment:
` i64` doesn't always fit into `f64`. Similar to `u64`, this needs two cases.

Reply:
Github display broke here. It's `impl Into<Any> for u32`.